src/networks/ensAbis/rsk.js

```diff
@@ -0,0 +1,25 @@
+const RSKAbi = [
+  {
+    constant: true,
+    inputs: [{ name: 'node', type: 'bytes32' }],
+    name: 'addr',
+    outputs: [{ name: '', type: 'address' }],
+    payable: false,
+    stateMutability: 'view',
+    type: 'function'
+  },
+  {
+    constant: false,
+    inputs: [
+      { name: 'node', type: 'bytes32' },
+      { name: 'addrValue', type: 'address' }
+    ],
+    name: 'setAddr',
+    outputs: [],
+    payable: false,
+    stateMutability: 'nonpayable',
+    type: 'function'
+  }
+];
+
+export default RSKAbi;
```

Comment:
@gamalielhere are we using these ENS ABI files anywhere? if not lets remove it from all networks!

Reply:
I was planning on using them for the ens registrar dapps. We can probably remove it for now.

Reply:
Yeah we can take it out, I don't think it is necessary :)